README.md

```diff
@@ -428,6 +428,54 @@ in Step 1 of the
 [Set up a Conjur OSS environment with persistence](#set-up-a-conjur-oss-environment-with-persistence)
 section above.
 
+### Adding or Modifying Container Environment Variables
+
+This section describes the process of either adding or modifying  environment variables in a running container. This process only requires stopping the container in question for a short time, while the rest of the system continues running.
+
+In the example below, the process will add an environment variable `CONJUR_LOG_LEVEL=debug` to the `conjur` service container.
+
+1. Add or modify environment variables in `docker-compose.yml`
+
+   `docker-compose.yml` is used to define the Conjur OSS system. Additions and modifications to environment variables are made in the `environment` configuration of the desired service, and are of the form:
+
+   ```
+   version: '3'
+   services:
+
+     conjur:
+
+       environment:
+         CONJUR_LOG_LEVEL: debug
+   ```
+
+1. Stop the container
+
+   ```
+   docker-compose stop conjur
+   ```
+
+1. Restart the container
+
+   ```
+   docker-compose up -d --no-deps conjur
+   ```
+
+   The service container now contains the environment variables defined in `docker-compose.yml`.
+
+1. Verify that the desired environment variables now belong to the  container
+
+   Run the following:
+
+   ```
+   docker-compose exec conjur env | grep CONJUR_LOG_LEVEL
```

Comment:
What about `docker-compose exec conjur printenv CONJUR_LOG_LEVEL` ? That way we can avoid the reader doing additional parsing when `env | grep CONJUR_LOG_LEVEL` returns more than one match.